sdk/python/testvectors/BlockFactory.py

```diff
@@ -0,0 +1,77 @@
+from symbolchain import sc
```

Comment:
not sure but might be more consistent to keep under vectors (similar to JS structure)

Reply:
there was command earlier, how will the imports inside testvectors dir look like?